Docker/Configuration/Adapter.php

```diff
@@ -142,7 +142,13 @@ public function writeToFile($file)
             }
         } elseif ($this->getFormat() == 'json') {
             $encoder = new JsonEncoder();
-            $serialized = $encoder->encode($this->getConfig(), $encoder::FORMAT, ['json_encode_options' => JSON_PRETTY_PRINT]);
+            $config = $this->getConfig();
+            foreach (['storage', 'parameters', 'image_parameters', 'authorization'] as $objectProp) {
```

Comment:
Co tohle vyjmout ven do vlastní metody a testovat jen že z `["storage" => []]` se stane `["storage" => \stdClass()]` místo testování celýho souboru?

Reply:
no ale ono se prave ze ["storage" => []] nestane ["storage" => \stdClass()] jenom se to tak zapise, docker runner ocekava vsude asociativni pole

Reply:
však jasně, bude to jen nějakej výstupní filtr